src/api/app/policies/token/rss_policy.rb

```diff
@@ -1,6 +1 @@
-class Token::RssPolicy < TokenPolicy
-  # TODO: when trigger_workflow is rolled out, remove the create? method
-  def create?
```

Comment:
`Token::RssPolicy#create?` is the same as `TokenPolicy#create?` after the changes in [the second one](https://github.com/openSUSE/open-build-service/pull/13810/commits/eb2754812ed1405de2b0f64a2dc68aa0b8c32701#diff-f156eef00f70113b2298c238401790daaed323194494afc4e29093d3355774f6L34).